diff_test.go

```diff
@@ -64,4 +64,4 @@ func TestDiffCmd(t *testing.T) {
 		}()
 	}
 	wg.Wait()
-}
+}
```

Comment:
```suggestion
}

```

@haya14busa as requested

Reply:
> We could think of inserting new blank line at the end alternatively like the above, but it's obviously different suggestion and unintended behabior.
>
> https://github.com/reviewdog/reviewdog/issues/1927#issuecomment-2633099908

This suggestion adds a new blank line, instead of adding EOF new line.

Reply:
If I commit it, I'm fairly certain it will add a EOF newline, even though it appears to be a blank line here